src/modules/clearAll.js

```diff
@@ -1,8 +1,20 @@
+/* eslint-disable no-restricted-globals */
+
 const clearBtn = document.querySelector('.clear-all');
 
-clearBtn.addEventListener('click', () => {
-  localStorage.clear();
-  window.location.reload();
-});
+const clearCompleted = () => {
+  const grab = JSON.parse(localStorage.getItem('tasks')) || [];
+
+  clearBtn.addEventListener('click', () => {
+    const tasks = grab.filter((tsk) => tsk.completed === false);
+
+    tasks.forEach((task, index) => {
+      task.index = index + 1;
+    });
+
+    localStorage.setItem('tasks', JSON.stringify(tasks));
+    location.reload();
+  });
```

Comment:
- [ OPTIONAL ]Your application is working fine but you don't have to reload the page to get the UI updated since your application has to be interactive so make sure that you create a function to Update the UI.